ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -0,0 +1,156 @@
+hideall "--reset--"
+hideall "--sync--"
+hideall "--start--"
+
+0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+
+
+###Phantom Ray
+#-ii F7D
+
+0 "Start"
+0.0 "--start--" sync /Exhibit level III will be sealed off/ window 0,1
+10.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
+16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+21.4 "Double Sever 1" sync /:Phantom Ray:F7B:/
+25.6 "Double Sever 2" sync /:Phantom Ray:F7C:/
+27.4 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+
+41.5 "Rapid Sever" sync /:Phantom Ray:F7A:/
+47.7 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+52.5 "Double Sever 1" sync /:Phantom Ray:F7B:/
+56.5 "Double Sever 2" sync /:Phantom Ray:F7C:/
+58.3 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ window 8,10 jump 27.4
+
+72.4 "Rapid Sever" sync /:Phantom Ray:F7A:/ jump 41.5
```

Comment:
Does this need a jump too? Or does the 58.3 cover it?

Reply:
I... think this was because of missed windows... somewhere? Might have been "let's just make SURE this jump happens by adding a backup". I'm not attached to it, so if you prefer to have only one jump per loop, I'm happy to remove it.

Reply:
Yeah, I think I'd prefer one jump per loop to make the timeline easier to read.  I think if I'm worried about missed windows, I'll just make the window larger on the loop jump.